### PR TITLE
pikopixel: init at 1.0-b9e

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

I'm having some problems with the application: the auxiliary windows constantly appear and disappear when clicking. I'm not sure if that's due to my window manager setup, so it'd be nice if someone with gnome/kde/something else tested it.
EDIT: I tested it on GNOME, and there are no issues, so it's just my setup.

Finally, I have no idea how to build this for darwin, and ofborg is down, so any help for that platform is appreciated. Is it just a matter of adding `xcbuild` to `nativeBuildInputs`?

### CODE REVIEW

File: pkgs/desktops/gnustep/default.nix [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
   self = rec {
-    stdenv = pkgs.clangStdenv;
+    stdenv = llvmPackages.stdenv;
 
     gsmakeDerivation = callPackage ./make/gsmakeDerivation.nix {};
     gorm = callPackage ./gorm {};
     projectcenter = callPackage ./projectcenter {};
     system_preferences = callPackage ./systempreferences {};
-    libobjc = callPackage ./libobjc2 {
-      stdenv = if stdenv.cc.isClang then llvmPackages_6.stdenv else stdenv;
```
Comment (fgaz) [inline pkgs/desktops/gnustep/default.nix:14]: why was this needed?

Comment (SuperSandro2000) [reply pkgs/desktops/gnustep/default.nix:14]: Seems like an old leftover to me.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106102` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>openvpn-auth-ldap</li>
    <li>pikopixel</li>
  </ul>
</details>
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>gnustep.libobjc</li>
    <li>gnustep.make</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gnustep.stdenv</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 106102` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>openvpn-auth-ldap: log <a href="https://termbin.com/gg7a">https://termbin.com/gg7a</a></li>
  </ul>
</details>
<details>
<summary>
15 packages built:
</summary>
<ul>
<li>bookworm</li>
<li>gnustep.back</li>
<li>gnustep.base</li>
<li>gnustep.gorm</li>
<li>gnustep.gui</li>
<li>gnustep.gworkspace</li>
<li>gnustep.libobjc</li>
<li>gnustep.make</li>
<li>gnustep.projectcenter</li>
<li>gnustep.system_preferences</li>
<li>hopper</li>
<li>pikopixel</li>
<li>sogo</li>
<li>sope</li>
<li>unar</li>
</ul>
</details>

**Comment by fgaz** [comment]
openvpn-auth-ldap is not a new failure: https://hydra.nixos.org/build/135254974

**Comment by SuperSandro2000** [comment]
> openvpn-auth-ldap is not a new failure: [hydra.nixos.org/build/135254974](https://hydra.nixos.org/build/135254974)

The check was faulty. On new runs it show the correct.
